data/words/famous_inspirational-quotes.json

```diff
@@ -0,0 +1,1311 @@
+           "from":"Emil Motycka"
+        },
+        {  
+           "text":"I honestly think it is better to be a failure at something you love than to be a success at something you hate.",
+           "from":"George Burns"
+        },
+        {  
+           "text":"Leaders don’t force people to follow, they invite them on a journey.",
+           "from":"Charles S. Lauer"
+        },
+        {  
+           "text":"Example is not the main thing in influencing other people; it’s the only thing.",
+           "from":"Abraham Lincoln"
+        },
+        {  
+           "text":"Leadership is doing what is right when no one is watching.",
+           "from":"George Van Valkenburg"
+        },
+        {  
+           "text":"Leadership is the art of getting someone else to do something you want done because he wants to do it.",
+           "from":"Dwight D. Eisenhower"
+        },
+        {  
+           "text":"The difference between a boss and a leader: a boss says, ‘Go!’ -a leader says, ‘Let’s go!’.",
+           "from":"E. M. Kelly"
+        },
+        {  
+           "text":"I am more afraid of an army of one hundred sheep led by a lion than an army of one hundred lions led by a sheep.",
+           "from":"Charles Maurice"
+        },
+        {  
+           "text":"The whole problem with the world is that fools and fanatics are always so certain of themselves, but wiser people so full of doubts.",
+           "from":"Bertrand Russell"
+        },
+        {  
+           "text":"We are what we repeatedly do. Excellence, then, is not an act, but a habit.",
+           "from":"Aristotle"
+        },
+        {  
+           "text":"Cannibals prefer those who have no spines.",
+           "from":"Stanislaw Lem"
+        },
+        {  
+           "text":"A ship in harbor is safe. But that’s now what ships are built for.",
+           "from":"William Shedd"
+        },
+        {  
+           "text":"If one does not know to which port one is sailing, no wind is favorable.",
+           "from":"Lucius Annaeus Seneca"
+        },
+        {  
+           "text":"You miss 100% of the shots you don’t take.",
+           "from":"Wayne Gretzky"
+        },
+        {  
+           "text":"I’m not a businessman. I’m a business, man.",
+           "from":"Jay-Z"
+        },
+        {  
+           "text":"The vision must be followed by the venture. It is not enough to stare up the steps – we must step up the stairs.",
+           "from":"Vance Hayner"
+        },
+        {  
+           "text":"Do not wait to strike until the iron is hot; but make it hot by striking.",
+           "from":"William B. Sprague"
+        },
+        {  
+           "text":"It’s easier to ask forgiveness than it is to get permission.",
+           "from":"Grace Hopper"
+        },
+        {  
+           "text":"Twenty years from now you will be more disappointed by the things that you didn’t do than by the ones you did do.",
+           "from":"Mark Twain"
```

Comment:
> Quote Investigator: QI will be unable to tell you where to find this passage in the works of Twain because he never wrote it. Yet, the words are regularly credited to him. For example, the April 20, 1998 issue of The New Yorker magazine printed a vibrant full page advertisement depicting an ocean scene that prominently featured a version of this saying with the label “attributed to Mark Twain”. 

https://quoteinvestigator.com/2011/09/29/you-did/